### ISSUE TITLE
Remove aria-controls dependency on button

### ISSUE THREAD

**Issue by ianmcburnie** [ianmcburnie/jquery-dialog#1]
aria-controls doesn't really make sense on the button that opens a dialog, because the dialog is not displayed or visible, therefore AT cannot provide a shortcut to it. Probably not too harmful for the time-being, but because I depend upon aria-controls value to identify the ID of the dialog, I will need to replace this functionality with a data attribute.
